src/main/java/uk/ac/ebi/embl/converter/fftogff3/FFToGff3Converter.java

```diff
@@ -25,16 +25,22 @@ public class FFToGff3Converter implements Converter {
 
     private static final Logger LOG = LoggerFactory.getLogger(FFToGff3Converter.class);
 
-    public void convert(BufferedReader reader, BufferedWriter writer) throws ConversionError {
-        try {
-            EmblEntryReader entryReader =
-                    new EmblEntryReader(reader, EmblEntryReader.Format.EMBL_FORMAT, "embl_reader", getReaderOptions());
+    public void convert(BufferedReader reader, BufferedWriter writer)
+            throws ReadException, WriteException, ValidationException {
+        EmblEntryReader entryReader =
+                new EmblEntryReader(reader, EmblEntryReader.Format.EMBL_FORMAT, "embl_reader", getReaderOptions());
 
-            GFF3FileFactory fftogff3 = new GFF3FileFactory();
-            GFF3File file = fftogff3.from(entryReader);
+        GFF3FileFactory fftogff3 = new GFF3FileFactory();
+        GFF3File file;
+        try {
+            file = fftogff3.from(entryReader);
+        } catch (IOException e) {
+            throw new ReadException(e);
+        }
+        try {
             file.writeGFF3String(writer);
         } catch (IOException e) {
-            throw new ConversionError("Error writing gff3 file", e);
+            throw new WriteException(e);
```

Comment:
We could add the try/catch in the fftogff3.from(entryReader); and file.writeGFF3String(writer); methods ond throw ReadException and WrightException respectively so that this method will be

```
.....
// Read embl entry
GFF3File file = fftogff3.from(entryReader);
// Write Gff3 file
file.writeGFF3String(writer);
......       
```